gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/GemFireCacheImpl.java

```diff
@@ -527,6 +554,10 @@
 
   private String vmIdRegionPath;
 
+  private final ThreadPoolExecutor diskStoreTaskPool;
```

Comment:
Please add a comment for the purpose of this pool in a comment. Will help new people looking into it.